buffer/src/test/java/io/netty/buffer/SlicedByteBufTest.java

```diff
@@ -138,16 +138,12 @@ public void testForEachByte2() {
         // Ignore for SlicedByteBuf
     }
 
-    @Test(expected = UnsupportedOperationException.class)
     @Override
     public void testDuplicateCapacityChange() {
-        super.testDuplicateCapacityChange();
     }
 
-    @Test(expected = UnsupportedOperationException.class)
     @Override
     public void testRetainedDuplicateCapacityChange() {
-        super.testRetainedDuplicateCapacityChange();
```

Comment:
@Scottmitch or we could also use `@Ignore("Sliced ByteBuf objects don't allow the capacity to change. So this test would fail and shouldn't be run")` . Maybe this would be even better... WDYT ?